experimental/language-generation/README.md

```diff
@@ -87,6 +87,47 @@ For NodeJS
     await turnContext.sendActivity(ActivityFactory.createActivity(lgEngine.evaluateTemplate("WordGameReply", { GameName = "MarcoPolo" } )));
 ```
 
+If you need to know the expand result of the evaluation of a template, `ExpandTemplate` is what you want.
```

Comment:
Move this section after multi lingual and grammar check stuff. It should be its own section - 
```
## Testing LG
To test a template, you would need to know all possible expansion results from evaluating that template. `ExpandTemplate` API gets you all possible expansions for a given template.

For C# ...

For JS ...

```